tests/common.py

```diff
@@ -22,6 +22,13 @@ def _reset_schema():
     rest.db.engine.execute('drop schema if exists public cascade;')
     rest.db.engine.execute('drop schema if exists disclosure cascade;')
     rest.db.engine.execute('create schema public;')
+    rest.db.engine.execute('create schema disclosure;')
+
+
+def _reset_schema_for_integration():
+    rest.db.engine.execute('drop schema if exists public cascade;')
+    rest.db.engine.execute('drop schema if exists disclosure cascade;')
+    rest.db.engine.execute('create schema public;')
```

Comment:
Oh wow, interesting, thanks for explaining that.  That does seem odd that `pg_dump` wouldn't account for the `public` schema but would all others; I'll try digging a bit too to see if I can uncover anything, but if not or it takes to long, good to know this is a working solution for us!